.github/ISSUE_TEMPLATE/proposal-template.md

```diff
@@ -8,60 +8,26 @@ assignees: crasskitty
 
 ---
 
-### Instructions:
-1. Fill out the proposal.
-2. Please select in the title if this is an Epic or a Task by deleting the word that does _NOT_ apply to what you are submitting.
-3. Assign yourself to this issue and tag in at least one other team member (other than @crasskitty--she's already assigned) to ask them to engage on your post!
-4. Engage on other people's proposals. Use the proposals as a way to engage with each other's ideas, poke holes in theories, and ask clarifying questions. Good luck and have fun!
-
-
-###  Common Questions:
-#### What is an Epic and what is a Task?
-- An Epic is an overarching goal that would include many Tasks, i.e. "Build a custom CRM." 
-- A Task is something that helps complete an Epic, i.e. "I need to import all of the companies in our network into this CRM."
-- Most proposals will be tasks.
-- _Note:_ Epics can span over a series of sprints!
-
-#### Where can I learn more about this process?
- - You can view the Notion document with full context [here](https://www.notion.so/nearprotocol/9-Day-Sprint-Cycle-a45f3324f3094a63887fa675f4a90292)
-
-
-### DELETE EVERYTHING ABOVE THIS LINE ↓
---------------------------------------------------------------------------------------------------------
 # Proposal for: [insert what you are proposing should be worked on in 10 words or fewer]
 
-## What will be accomplished by completing this and who will it impact?
-- (delete this line) describe what will be accomplished and who will be affected, in as much detail as you can.
-
-## Which OKR(s) do this relate to?
-- (delete this line) select all that apply by putting a `x` inside the brackets
+## What is this? 
+- 
 
-- [ ] 1. Increase the number of **successful** apps running on MainNet
-- [ ] 2.  Increase the number of **successful** developers deploying and improving their apps on TestNet
-- [ ] 3. Support the continued maintenance of the materials that will allow devs to self-serve and be **successful**
+## Who needs to be involved?
+-
 
-## What resources do you need to accomplish this?
-- (delete this line) describe what resources you think you might or will need to accomplish this.
+## How will we know this is done? (Definition of Done)
```

Comment:
```suggestion
## Tasks to complete
[ ] - 
```

What do you think of changing this to "Tasks to complete"?  I like having a list of checkboxes breaking down the steps and indicating when the work is done.